.github/workflows/ci.yml

```diff
@@ -4,14 +4,14 @@ on: [push, pull_request]
 
 jobs:
   run:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
 
     strategy:
       matrix:
-        php: [7.2, 7.3, 7.4]
-        extension: [':bcmath', ':gmp']
+        php: [7.2, 7.3, 7.4, 8.0]
+        extension: ['bcmath', 'gmp']
```

Comment:
Does this work? The reason why I disabled the extensions using `:bcmath` is because they are loaded by default.

Reply:
I've updated my PR with these needs. Is this ok to you?

Reply:
Hmm, I think we can revert back to the previous behavior. We don't need to specify which extension should be enabled since it is enabled by default.

Reply:
done